save-demo/src/main/kotlin/com/saveourtool/save/demo/controller/ManagementController.kt

```diff
@@ -87,6 +94,64 @@ class ManagementController(
         .flatMap {
             demoService.loadFileToStorage(organizationName, projectName, version, file)
         }
+        .map {
+            FileDto(
```

Comment:
`FileDto` supposes to store this key in database and use it as key of storage.
why do you need `FileDto` here? you have `ToolKey`

Reply:
For now with use `ToolKey`, will be changed in PR with db-storage integration

Reply:
Oh will leave as is as far as `ToolKey` is `save-demo` specific